### PR TITLE
media-video/mpv: verbump to 0.21.0

### PR DESCRIPTION
**Pull request by Coacher**
Package-Manager: portage-2.3.2

### CODE REVIEW

**Comment by Coacher** [comment]
CC @gentoo/proxy-maint

File: media-video/mpv/mpv-0.21.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+pkg_pretend() {
+	if [[ ${MERGE_TYPE} != "binary" ]] && ! tc-has-tls && use vaapi && use egl; then
+		die "Your compiler lacks C++11 TLS support. Use GCC>=4.8.0 or Clang>=3.3."
+	fi
+}
+
+src_prepare() {
+	cp "${DISTDIR}/waf-${WAF_PV}" "${S}"/waf || die
+	chmod +x "${S}"/waf || die
+	default_src_prepare
```
Comment (SoapGentoo) [inline media-video/mpv/mpv-0.21.0.ebuild]: `default`

**Comment by Coacher** [comment]
I've slightly updated this PR to check compiler features in pkg_setup() too.

**Comment by Coacher** [comment]
Thank you, @SoapGentoo.

**Pull request closed without merging**